src/RequestObjects.ts

```diff
@@ -139,12 +139,7 @@ export class OnboardWithThingIDRequest {
  * @prop {Object} metadata Key-value list to store within command definition.
  */
 export class PostCommandRequest {
-    public aliasActions: Array<AliasAction>;
     public issuer?: string;
-    public title?: string;
-    public description?: string;
-    public metadata?: Object;
-
```

Comment:
This might be lack of my knowledge for typescript but I appreciate if you answer this question.

Why we can remove above member fields? Also why we should leave issuer?